src/index.js

```diff
@@ -83,45 +136,60 @@ function visitCodeBlock(ast, vFile) {
  * If links have a title attribute called `mermaid:`, follow the link and
  * generate the graph. Replace the link with the link to the generated graph.
  *
- * @param {object} ast
- * @param {vFile} vFile
+ * @param {object}  ast
+ * @param {vFile}   vFile
  * @return {function}
  */
-function visitLink(ast, vFile) {
+function visitLink(ast, vFile, isSimple) {
+  if (isSimple) {
+    return visit(ast, 'link', (node, index, parent) => replaceLinkWithEmbedded(node, index, parent, vFile));
+  }
+
   return visit(ast, 'link', node => replaceUrlWithGraph(node, vFile));
 }
 
 /**
```

Comment:
Update comment to explain "if (isSimple)..."

Reply:
Good catch